aleph/settings.py

```diff
@@ -81,6 +81,8 @@
 OAUTH_AUTHORIZE_URL = env.get('ALEPH_OAUTH_AUTHORIZE_URL')
 OAUTH_UI_CALLBACK = env.get('ALEPH_OAUTH_UI_CALLBACK', '/oauth')
 
+# No authentication. Everyone is admin.
+SINGLE_USER = env.to_bool('SINGLE_USER')
```

Comment:
Ok, so I'd prefix the actual env var with `ALEPH_`, just not the variable derived from it in `settings` (like all the other vars in this file).